includes/class-migration.php

```diff
@@ -203,6 +199,10 @@ public static function maybe_migrate() {
 			}
 		}
 
+		// Always register schedules, to ensure they are registered even
+		// if the plugin is activated after the first activation.
```

Comment:
```suggestion
		// Ensure all required cron schedules are registered.
```
This might be all that's needed in terms of context?

Reading "ensure they are registered even if the plugin is activated after the first activation" makes me wonder what's happening there, what's going wrong, what needs fixing, if that makes sense?

It's probably fine for us to just keep this as a fallback. If it's to fix a bug that we've not investigated, we should investigate it and fix it 🙂

Reply:
That case have so many reasons! I also accidentally removed too many schedules, because of some were stuck! I would not assume a bug here! The logic is quite simple and I see nothing else that might delete a schedule.

Reply:
@obenland because of the context: haven't you seen the slack about that issue? I hoped so and didn't want to reference private chats!

Reply:
I see this similar to the default value of an option/meta! It can always happen that a db entry will be deleted (by cleanup plugins, manual db editing, backup/restore issues), so you have to build a system to compensate these cases.

Reply:
It didn't mention deactivations/re-activations from what I could tell?

Like I said, I don't mind the code change and having this as a fallback/insurance. It's more about the context for it if that makes sense?

Reply:
Let's just say we want it to make sure cron schedules are what they should be

Reply:
Ok, maybe I have misunderstood 'context' 😉

Reply:
I understand your point, but in this case we only have the context that one schedule was not existent!